aten/src/ATen/native/UnaryOps.cpp

```diff
@@ -310,6 +319,16 @@ Tensor& mvlgamma_(Tensor& self, int64_t p) {
     return result;                                                     \
   }
 
+// IMPLEMENT_UNARY_FLOATING_OP_* macros are used for ops which support 
```

Comment:
Per the above, point to the previous comment, not the PR.

Reply:
Done! (https://github.com/pytorch/pytorch/pull/33322/commits/c816f39f45369536c2ffb329756ae36405b21d64)